cns/restserver/api.go

```diff
@@ -919,11 +919,15 @@ func (service *HTTPRestService) getNetworkContainerByOrchestratorContext(w http.
 // If received  "GET": Return all NCs in CNS's state file to DNC in order to check if NC refresh is needed
 // If received "POST": Store all the NCs (from the request body that client sent) into CNS's state file
 func (service *HTTPRestService) getOrRefreshNetworkContainers(w http.ResponseWriter, r *http.Request) {
+	logger.Printf("[Azure CNS] NC refresh is initiated by DNC. The state file in CNS will get updated if it is lost.")
+
 	switch r.Method {
 	case http.MethodGet:
+		logger.Printf("[Azure CNS] DNC is attempting to check if the state file in CNS is intact.")
 		service.handleGetNetworkContainers(w)
 		return
 	case http.MethodPost:
+		logger.Printf("[Azure CNS] The state file in CNS is lost. CNS will store all the NCs sent from DNC into state file.")
```

Comment:
or, `POST to getOrRefreshNetworkContainers`

Reply:
Addressed.